dora/core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -6795,6 +6795,21 @@ public String toString() {
                   + "etcd cluster (e.g. http://localhost:2379,http://etcd1:2379)")
           .setScope(Scope.ALL)
           .build();
+  public static final PropertyKey ETCD_USERNAME =
+      stringBuilder(Name.ETCD_USERNAME)
+          .setDescription("User name used for EtcdMembership module to do worker"
+              + " registration. Make sure the given user has the full readwrite"
+              + " role permission on all keys with prefix '/'. "
+              + "Refer to etcd official site for authentication:"
+              + "https://etcd.io/docs/v3.2/op-guide/authentication/")
+          .setScope(Scope.ALL)
+          .build();
+  public static final PropertyKey ETCD_PASSWORD =
+      stringBuilder(Name.ETCD_PASSWORD)
+          .setDescription("User password used for EtcdMembership module to do worker"
+              + " registration")
```

Comment:
```suggestion
          .setDescription("User password used to connect to ETCD. Alluxio can use ETCD to store worker membership and mount table information.")
```

Reply:
actually this is in CE right, i will just say for etcd communication